components/centraldashboard/public/components/main-page.js

```diff
@@ -166,6 +166,34 @@ export class MainPage extends utilitiesMixin(PolymerElement) {
         this.$.welcomeUser.show();
     }
 
+    /**
```

Comment:
Why not use the declarative `text` and `opened` properties for paper-toast as opposed to imperatively opening/closing it?

https://www.webcomponents.org/element/@polymer/paper-toast/elements/paper-toast

Reply:
This one shows it for the delayed amount (which is 5s), opened I believe will keep it persisted.

Reply:
https://www.webcomponents.org/element/@polymer/paper-toast/elements/paper-toast#property-duration looks like it can be used to set the duration. Either way is appropriate but it's nice to use the property surface as it helps reduces the mock interactions that need to be tested.

Reply:
Fair enough, will change!

Reply:
So Jason, for this element, the text prop is already set, so is the duration. My point is that we need this to show after 500ms, and we never need to close this element.

`.show()` opens the element, waits `duration` long, and closes the element. This behavior may not make sense via props, since we set a state that then either get's changed or outdated in `duration` ms.

Reply:
My suggestion was to have properties on the main-page component that are passed down to the paper toast so that changing them affects the display behavior for paper toast.

Then you could just have this in the markup:
```
paper-toast#ErrorToast(opened='[[errorText]]', text='[[errorText]]', duration=0)
```
That at least handles the display part of it. Then all `closeToast` needs to do is clear the errorText property. You can even drop down to having a single paper-toast element with other props to control the style (error vs. information).